OPTIMIZATION: Storage, Capacity, and Prerequisites (#1118)
minio/docs: source/operations/install-deploy-manage/expand-minio-deployment.rst

While each pool is an independent set of servers with distinct :ref:`erasure sets <minio-ec-erasure-set>` for availability, the complete loss of one pool results in MinIO stopping I/O for all pools in the deployment.
Similarly, an erasure set which loses quorum in one pool represents data loss of objects stored in that set, regardless of the number of other erasure sets or pools.

> The new server pool does **not** need to use the same type or size of  hardware and software configuration as any existing server pool, though doing so may allow for simplified cluster management and more predictable performance across pools.

Reviewer: ```suggestion
The new server pool does **not** need to use the same type or size of hardware and software configuration as any existing server pool, though doing so may allow for simplified cluster management and more predictable performance across pools.
```

While each pool is an independent set of servers with distinct :ref:`erasure sets <minio-ec-erasure-set>` for availability, the complete loss of one pool results in MinIO stopping I/O for all pools in the deployment.
Similarly, an erasure set which loses quorum in one pool represents data loss of objects stored in that set, regardless of the number of other erasure sets or pools.

> The new server pool does **not** need to be substantially similar in hardware and software configuration to any existing server pool, though this may allow for simplified cluster management and more predictable performance across pools.

Reviewer: This may lead people to think an expansion pool can have drives that mix type or size.
```suggestion
The new server pool does **not** need to use the same type or size of  hardware and software configuration as any existing server pool, though doing so may allow for simplified cluster management and more predictable performance across pools.
All drives in the new pool **should** be of the same type and size within the new pool.
```

listed value is at least ``2 x EC:N``, the pool supports the deployment's
erasure parity settings.

Time Synchronization
~~~~~~~~~~~~~~~~~~~~

Multi-node systems must maintain synchronized time and date to maintain stable internode operations and interactions.
Make sure all nodes sync to the same time server regularly.
Operating systems vary for methods used to synchronize time and date, such as with ``ntp``, ``timedatectl``, or ``timesyncd``.

Check the documentation for your operating system for how to set up and maintain accurate and identical system clock times across nodes.

Back Up Cluster Settings First
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Use the :mc:`mc admin cluster bucket export` and :mc:`mc admin cluster iam export` commands to take a snapshot of the bucket metadata and IAM configurations respectively prior to starting decommissioning.

> You can use these snapshots to restore bucket/IAM settings to recover from user or process errors as necessary.

Reviewer: ```suggestion
You can use these snapshots to restore :ref:`bucket <minio-mc-admin-cluster-bucket-import>` and :ref:`IAM <minio-mc-admin-cluster-iam-import>` settings to recover from user or process errors as necessary.
```
